Physical_Engine/Query/HasMergeablePropertiesWith.cs

```diff
@@ -128,6 +128,29 @@ public static bool HasMergeablePropertiesWith(this IOpening element, IOpening ot
 
         /***************************************************/
 
+        [Description("Evaluates if the two PadFoundation non-geometrical data is equal to the point that they could be merged into one object.")]
+        [Input("element", "An PadFoundation to compare the properties of with an other PadFoundation.")]
+        [Input("other", "The PadFoundation to compare with the other PadFoundation.")]
+        [Output("equal", "True if the PadFoundation non-geometrical property is equal to the point that they could be merged into one object.")]
+        public static bool HasMergeablePropertiesWith(this PadFoundation element, PadFoundation other)
+        {
+            if (element.IsNull() || other.IsNull())
+                return false; //If either is null, then it can probably can't have its properties merged
+
+            if (element.Construction == other.Construction)
+                return true;
+
+            if (element.Construction.IsNull() || other.Construction.IsNull())
```

Comment:
What if both constructions are null? Isn't it valid to merge properties then?

Reply:
Is that not covered by L140?

Reply:
of course it is, me blind